manifests/h/HDOS/HDOSLauncher/262/HDOS.HDOSLauncher.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: HDOS.HDOSLauncher
+PackageVersion: 262

```

```suggestion
PackageVersion: 1.0.8
```
This should match control panel, or AppsAndFeaturesEntries should be added